keps/sig-apps/3998-job-success-completion-policy/README.md

```diff
@@ -381,214 +382,6 @@ To avoid the above conflicts, terminating policies are evaluated the first befor
 This means that the terminating policies are respected rather than the successPolicies,
 if the Job doesn't have the `FailureTarget` or `SuccessCriteriaMet` conditions yet.
 
-### Future Work
```

Comment:
Since we move it out of the scope I would suggest to remove the "Optional Second Alpha" section from Graduation Criteria.

Reply:
You're right.